src/components/edit/EarthEngineDialog.js

```diff
@@ -70,10 +69,20 @@ const datasets = {
 
 const styles = {
     ...layerDialogStyles,
+    flexFull: {
+        ...layerDialogStyles.flexFull,
+        marginBottom: 12,
+    },
     legend: {
-        borderLeft: '12px solid #fff',
-        borderRight: '12px solid #fff',
+        padding: '0 16px 16px 32px',
+        margin: 0,
+        marginLeft: -32,
     },
+    // Unused?
```

Comment:
@turban I left this as as comment for now, feel free to have me remove if if you think it's safe to remove.

Reply:
The borders are adding extra margin to the "columns" - see screenshot here: https://github.com/dhis2/maps-app/pull/21#issuecomment-423465031

Reply:
Ref. https://stackoverflow.com/questions/20626685/better-way-to-set-distance-between-flexbox-items
Open for other ways to do the layout positioning :-)